src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -163,16 +164,16 @@ class RsErrorAnnotator : RsAnnotatorBase(), HighlightRangeExtension {
     }
 
     private fun checkReferenceIsPublic(ref: RsReferenceElement, o: PsiElement, holder: AnnotationHolder) {
-        val element = ref.reference.resolve() as? RsVisible ?: return
+        val element = ref.reference.resolve() as? RsVisibilityOwner ?: return
```

Comment:
Looks like it's wrong assumption (e.g. RsFile is RsVisible but not RsVisibilityOwner)